### PR TITLE
Update 4.11 20220707

### PR DESCRIPTION
**Pull request by swatisehgal**
Manual cherry-picks of the following PRs to release-4.11 branch:
- https://github.com/kubernetes-sigs/scheduler-plugins/pull/344
- https://github.com/kubernetes-sigs/scheduler-plugins/pull/377
- https://github.com/kubernetes-sigs/scheduler-plugins/pull/340

### CODE REVIEW

File: test/integration/elasticquota_controller_test.go [hunk 1]
```diff
@@ -37,9 +37,9 @@ import (
 	fwkruntime "k8s.io/kubernetes/pkg/scheduler/framework/runtime"
 	st "k8s.io/kubernetes/pkg/scheduler/testing"
 
-	"sigs.k8s.io/scheduler-plugins/pkg/apis/scheduling"
-	"sigs.k8s.io/scheduler-plugins/pkg/apis/scheduling/v1alpha1"
-	schedv1alpha1 "sigs.k8s.io/scheduler-plugins/pkg/apis/scheduling/v1alpha1"
+	"sigs.k8s.io/scheduler-plugins/apis/scheduling"
```
Comment (ffromani) [inline test/integration/elasticquota_controller_test.go]: looks OK. I wonder: how costly is to cherry-pick the commit(s) that fixed this u/s?

**Review by ffromani — COMMENTED** [review]
thanks, looks good. Question inside: what's the tradeoff wrt cherry-picking more u/s commits?

File: test/integration/elasticquota_controller_test.go (hunk 1 above)
Comment (swatisehgal) [reply test/integration/elasticquota_controller_test.go]: We could certainly do that. Here is the commit https://github.com/kubernetes-sigs/scheduler-plugins/pull/340/commits/b5ea0f1f188e37f770868c7a74a9ae91107dd4e1. Would you prefer it to be done as part of this PR?

Comment (ffromani) [reply test/integration/elasticquota_controller_test.go]: > We could certainly do that. Here is the commit [kubernetes-sigs@b5ea0f1](https://github.com/kubernetes-sigs/scheduler-plugins/commit/b5ea0f1f188e37f770868c7a74a9ae91107dd4e1). Would you prefer it to be done as part of this PR?

if cherry-picking this commit happens cleanly and if is not too much hassle, I'd prefer to cherry-pick it and keep the history as close as possible. Most of these changes are not compiled in in our artifacts, so it's almost zero extra risk.

Comment (swatisehgal) [reply test/integration/elasticquota_controller_test.go]: Ack, on it.

**Comment by ffromani** [comment]
/lgtm
thanks for the update!

**Pull request merged**